sphinx/themes/basic/theme.conf

```diff
@@ -7,3 +7,4 @@ pygments_style = none
 nosidebar = false
 sidebarwidth = 230
 navigation_with_keys = False
+canonical_url =
```

Comment:
Why do you wan to this as an option of HTML themes?

Reply:
Apposed to adding it to the HTML builder?

Reply:
Sorry for late response. Yes, I meant that. I think the responsibility of HTML theme is look and feel, and canonical URL is not.

Reply:
Ok, I will look into that.